contributing.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to Ayria Desktop
+_**Thank you for your interest in contributing!** This document will outline the basics of where to start if you wish to contribute to the project. There are many ways to help us out and and we appreciate all of them. We look forward to **your contribution**!_
+
+## Communication
+
+### Chat
+The quickest and most open way to **communicate with the Ayria team** is on our [**Discord channel**](https://discord.gg/Jg8NZDK).
+
+### GitHub Issues
+A bit more formal way of communication with fellow Ayria members, but a little less quick and convenient like the chat. Submit an issue when you run into problems or just would like to discuss a certain topic, be it _features, code style, code inconsistencies, minor changes and fixes, etc._
+
+### Pull Requests
+It’s fine to just submit a small pull request without first making an issue or asking in the chat unless if it’s a significant change that will require **planning and reviewing**. Also see [Creating a Pull Request](#creating-a-pr) and [Git Style Guidelines](#git-style-guidelines).
+
+## Possible Contributions
+
```

Comment:
* Pipe IO for fetching account information etc..

Reply:
* Callback for whitelisting manually installed plugins [Bootstrap_cpp ref](https://github.com/AyriaPublic/Bootstrapmodule_cpp/blob/master/Source/Bootstrapper/Pluginloader.cpp#L42-L48)

Reply:
That's not something that can be worked on right now, right? Same for package management.

Reply:
I mean, a system for opening a pipe and reading/writing data can be done at any time. Fetching the actual data requested however.. =P